feed/feed_test.go

```diff
@@ -399,6 +423,29 @@ func TestPreScoredGenerator(t *testing.T) {
 			},
 			expectedPosts: []string{nsfwArtPost, nsfwLabelledPost},
 		},
+		{
+			name: "all videos",
+			opts: preScoredGeneratorOpts{
+				Alg: "classic",
+				generatorOpts: generatorOpts{
+					Hashtags: []string{},
+					HasMedia: tristate.True,
+				},
+			},
+			expectedPosts: []string{videoPost, nsfwVideoPost},
+		},
+		{
+			name: "tagged videos nsfw",
+			opts: preScoredGeneratorOpts{
+				Alg: "classic",
+				generatorOpts: generatorOpts{
+					Hashtags: []string{"art", "furryart"},
```

Comment:
Hmm should these hash tags be here ?

Reply:
Yeah, I wanted to test videos tagged with `#furryart`.